### PR TITLE
Fix password validation

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Fix password validation in all solutions

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1062

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Comment by maayarosama** [comment]
Password field doesn't accept special characters which shouldn't be the case
password

File: packages/playground/src/weblets/profile_manager.vue [hunk 1]
```diff
@@ -185,6 +185,7 @@
                   :rules="[
                     validators.required('Password is required.'),
                     validators.minLength('Password must be at least 6 characters.', 6),
+                    validators.isAlphanumeric('Password should consist of letters and numbers only.'),
```
Comment (AhmedHanafy725) [inline packages/playground/src/weblets/profile_manager.vue:188]: not sure if we should change the rules of the password of the wallet connector as it's frequently used by the user and that may prevent the login for them

Comment (zaelgohary) [reply packages/playground/src/weblets/profile_manager.vue]: I reverted the previous changes and verified on white spaces only. should I remove the changes from the profile manager?

**Comment by zaelgohary** [comment]
@maayarosama I added a new validation pattern which checks if the string contains a space or not.

File: packages/playground/src/weblets/profile_manager.vue (hunk 1 above)
Comment (AhmedHanafy725) [reply packages/playground/src/weblets/profile_manager.vue]: yes, it is used only for encrypting the mnemonic in local storage and works fine with spaces

Comment (zaelgohary) [reply packages/playground/src/weblets/profile_manager.vue]: Done.

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**